### ISSUE TITLE
time zone problem in events in FAIRDOMHub

### ISSUE THREAD

**Issue by wittigue** [seek4science/seek#989]
after creating an event (e.g. https://fairdomhub.org/events/302) with start/end time definition a time difference of 2 hours is shown compared to the definition

**Comment by alaninmcr** [comment]
Is that because it is showing the time in UTC and not CEST?

Alan

________________________________________
From: wittigue ***@***.***>
Sent: 28 April 2022 19:16
To: seek4science/seek
Cc: Subscribed
Subject: [seek4science/seek] time zone problem in events in FAIRDOMHub (Issue #989)

after creating an event (e.g. https://fairdomhub.org/events/302) with start/end time definition a time difference of 2 hours is shown compared to the definition

**Comment by wittigue** [comment]
yes, it could be UTC but it's not written somewhere. Or it should be possible to select a time zone.

**Comment by floradanna** [comment]
#640 there was already an issue about this

**Comment by alaninmcr** [comment]
I've flagged the issue as a bug rather than just an enhancement.

________________________________________
